source/ui/Components/Register/DefaultRegister.qml

```diff
@@ -95,4 +96,63 @@ TextField {
         return registerContentString;
     }
 
+    style: whiteRectangle
+
+    //used for highlighting
+    Component{
+        id: styleChanged
+        TextFieldStyle{
+            background:  Loader {
+                id: loader
+                focus: false
+                sourceComponent: glowEffect
+            }
+
+            Component {
+                id: glowEffect
+                Item{
+                    Rectangle{
+                        id: rect
+                        x: registerTextField.x; y: registerTextField.y
```

Comment:
I'd suggest to put these in two separate lines to have a uniform style among our files; but that's just a really minor thing.